src/thunks/postProject.js

```diff
@@ -7,10 +7,10 @@ export const postProject = (name) => {
       dispatch(setLoading(true));
       const newProjectId = await fetchData('/api/v1/projects', 'POST', { name });
       dispatch(addProject({ id: newProjectId[0], name }));
-      return newProjectId[0]
+      dispatch(setLoading(false));
+      return newProjectId[0];
```

Comment:
We were never hitting our dispatch(setLoading(false)) because of the return. We did this so we could get the id straight from here, but it's probably better if we get the id from redux store instead and take out the return here. (line 20, ControlForm)

Reply:
I updated this functionality in my latest PR so we get back the entire object so this is a little different now.

Reply:
Ok no problem, we can fix tests again after your PR